replies/main_test.go

```diff
@@ -47,10 +47,14 @@ type testsession struct {
 	messages map[string]*strings.Builder
 }
 
-func (s *testsession) AddHandler(handler interface{}) func() {
+func (s *testsession) AddHandler(handler any) func() {
```

Comment:
```suggestion
func (s *testsession) AddHandler() func() {
```
unused parameter, believe this will work (?)